riscv64/build_finalize.sh

```diff
@@ -5,7 +5,11 @@ apt-get update
 
 DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y \
     openssh-client libslirp-dev libfdt-dev libglib2.0-dev libssl-dev \
-    libpixman-1-dev netcat
+    libpixman-1-dev
+
+# Modify fstab to mount `tmpfs` during boot
+# See: https://wiki.qemu.org/Documentation/9p_root_fs#Let's_start_the_Installation
+echo 'tmpfs /tmp tmpfs rw,nosuid,nodev,size=524288k,nr_inodes=204800 0 0' >> $ROOTFS_DIR/etc/fstab
```

Comment:
Why not `noatime,size=500M` like the linked documentation suggests? :o 

Also, in the commit message, let's state that various unittests explicitly expect `/tmp` to exist, and were failing on riscv because no tmpfs was mounted. The "failed to determine file size" example from the documentation is specific to `fstat` syscalls, which our unittests are not doing

Reply:
That's from the same documentation, below the section you quoted:

> Alternatively you could of course also configure it by adding an entry to /etc/fstab instead, e.g. something like:

> echo 'tmpfs /tmp tmpfs rw,nosuid,nodev,size=524288k,nr_inodes=204800 0 0' >> /etc/fstab

I will amend the commit message right now :)

Reply:
ahhh, my bad. Didn't read that far, haha. Looks good then :)